WordPressAuthenticator/Unified Auth/View Related/SiteAddressViewController.swift

```diff
@@ -29,12 +30,28 @@ final class SiteAddressViewController: LoginViewController {
                                   keyboardWillHideAction: #selector(handleKeyboardWillHide(_:)))
     }
 
+    /// Localize the "Continue" button
+    ///
     func localizePrimaryButton() {
         let primaryTitle = displayStrings.continueButtonTitle
         submitButton?.setTitle(primaryTitle, for: .normal)
         submitButton?.setTitle(primaryTitle, for: .highlighted)
     }
 
+    /// Registers all of the available TableViewCells
+    ///
+    func registerTableViewCells() {
+        let cells = [
+            InstructionTableViewCell.reuseIdentifier: InstructionTableViewCell.loadNib(),
+        ]
+
+        for (reuseIdentifier, nib) in cells {
+            tableView.register(nib, forCellReuseIdentifier: reuseIdentifier)
+        }
```

Comment:
Is this better? I made it slightly less verbose but still readable.

Reply:
It's up to you. It's just that a `for` loop isn't necessary at all if `forEach` is used. But either way works.

Reply:
`forEach` isn't more performant than `for in`. I'm going to keep it because I like the readability.